packages/core/src/components/forms/_form-group.scss

```diff
@@ -10,7 +10,7 @@ Form groups
 Markup:
 <div class="#{$ns}-form-group">
   <label class="#{$ns}-label" for="example-form-group-input-a">
-    Label A
+    <span class="#{$ns}-label-text">Label A</span>
```

Comment:
why not just use `<strong>` tags if all you want is bold? it's both semantic and requires no additional styles or classes.

Reply:
cause if we change this in the future, we'll also have to change the tag. poor maintanability

Reply:
but that tag isn't going anywhere! it's the semantic browser way of saying "make this text stronger," which is exactly what you're trying to do here.

Reply:
please try it. i'm quite confident it'll work beautifully and we won't have to add another `-text` class for one style rule.

Reply:
what I meant is that I don't like tying a visual to a specific tag. if in the future I want to get rid of bold and do another visual style, I'll have to change the tag because `<strong>` will no longer make sense.

did it anyway, kind of a bummer tho cause I still need to override the `font-weight` of the `<strong>` (default is 700, we want 600)